xblock/fields.py

```diff
@@ -819,6 +819,10 @@ class List(JSONField):
     _default = []
 
     def from_json(self, value):
+        # Empty strings are treated as empty lists
+        if isinstance(value, basestring) and not value:
+            value = self._default
```

Comment:
Should this be `value = []` instead?

i.e. if in my code I put something like `some_field = List(default=['a', 'b'])`, then I would expect `<block some_field=""/>` to be explicitly not using the default (`<block/>` which is the same as `<block some_field='["a", "b"]'>`). Though I guess that's rare and `some_field=""` is technically malformed anyways, so I guess this isn't a big deal.

Reply:
Fixed with https://github.com/edx/XBlock/pull/419/commits/8db4ce80fac28c33604a7b96fa3adbaa2c64bcb5#diff-cef7a9b520121ab3e6f4e83a4ca297c7R826